### DISCUSSION TITLE
The V100 SXM2 system has become surprisingly affordable

### DISCUSSION THREAD

**Discussion by bobqianic** [ggml-org/llama.cpp#5229] [General]
### There's a large number of brand-new Gigabyte T181-G20 barebone servers available in Hong Kong, **each selling for around £120**.

![Image_20240130235127](https://github.com/ggerganov/llama.cpp/assets/129547291/08dd6303-355a-429e-84f3-b2b1440d572d)

**Comment by ggerganov** [comment]
We should plug them to `ggml-ci`

**Reply by bobqianic** [reply]
The only issue I've noticed is that these servers lack a built-in power supply. They need an OCP v1.0 server rack.

**Comment by sorasoras** [comment]
Where can I bought them LOL

**Reply by bobqianic** [reply]
If you live in mainland China, you should be able to find them on Xianyu, Taobao very soon. They are currently going through customs. If you live elsewhere (Hong Kong or places outside mainland China), you need to find a logistics company yourself to transport them from Hong Kong to your location.

**Comment by fgdfgfthgr-fox** [comment]
Just checked its sale on Xianyu, it doesn't seem like a strong competitor to 2080ti 22G, which has a larger VRAM and better compatibility.

**Reply by bobqianic** [reply]
When it comes to inference tasks, the 2080ti with 22GB is the superior choice. However, for substantial training, the V100 with 16GB significantly outperforms the 2080ti. This is mainly because the V100 offers 125 TFLOPs of FP16 computational power compared to the 50 TFLOPs in the 2080ti. Additionally, the V100 benefits from a higher bandwidth interconnection due to NVLINK, which is especially advantageous when scaling up for training across multiple GPUs.

**Reply by richginsberg** [reply]
Do you have skus for 2gb ddr6 BGA chips? Want to mod a bunch of 2080ti.

**Reply by bobqianic** [reply]
`K4ZAF325BM-HC16`

You can easily find them on the market.

**Reply by FreemanGPT** [reply]
did anyone ever see them for close to 150 dollar listed? if yes where and in which quantity. How did @bobqianic know that they were 150 Dollar before they got listed. I want to buy a bunch of them if somehow possible but they don't seem to be available. So I wonder if they were ever?

**Comment by ricoelmostico** [comment]
Are they still on sale?
Do you have a link?

**Reply by BullshiArtist** [reply]
you can buy it on a second hand planform called "Xianyu" or Taobao and the price is $847 and add $141 with the power supply

**Comment by sprappcom** [comment]
@bobqianic anyone tried buying this? is it worth the buy? i always wonder if buying p40 is better or this one is.

**Comment by FreemanGPT** [comment]
@bobqianic Are they still available? I can't find the links.
